nilearn/datasets/data/seitzman_2018/merge_seitzman_files.py

```diff
@@ -0,0 +1,30 @@
+"""This script takes original files from Seitzman atlas
+    - ROIs_300inVol_MNI_allInfo.txt
+    - ROIs_anatomicalLabels.txt
+and creates one nicely formatted csv file (seitzman_2018_rois.csv) with
+coordinates, network and region names
+"""
+import pandas as pd
```

Comment:
I'm not sure that I like having this code in the nilearn tree, as we cannot test it under CI (because it requires pandas).

Reply:
Actually, my point is rather that this file, as well as the two txt files, should be moved out of the nilearn package, and should not be shipped with nilearn. Rather, they should be in the repository for developers to use them.

Maybe we could add a directory "dev_utils" at the base of the source tree, and move them in a subdirectory there, with a clear readme.

Reply:
I tried anyway :) It wasn't so tricky, once I found the `recfunctions` module.

The two original input text files now reside in the `data` directory and are read directly by the fetcher. The install issue should now be fixed as well.